packages/beacon-node/src/chain/bls/jobItem.ts

```diff
@@ -73,12 +74,31 @@ export function jobItemWorkReq(job: JobQueueItem, format: PointFormat, metrics:
       const signatures = job.sets.map((set) => bls.Signature.fromBytes(set.signature, CoordType.affine, true));
       timer?.();
 
+      // adding verification randomness is swig specific. must not attempt with herumi until
+      // @chainsafe/bls is updated to support it with herumi
+      if (job.opts.addVerificationRandomness) {
```

Comment:
do we add verification randomess with napi?

Reply:
Yes. And it's turned on for feat2

Reply:
I thought we should be able to add randomess factor as native side as stated in https://github.com/ChainSafe/lodestar/issues/6462#issuecomment-1961069290